addons/test_discuss_full/tests/test_performance.py

```diff
@@ -519,6 +526,7 @@ def _get_init_messaging_result(self):
                                     'id': self.users[0].id,
                                     'isInternalUser': True,
                                 },
+                                'user_livechat_username': False,
```

Comment:
The idea would be to sent either the `name` or the `user_livechat_username`, never both.